docs/platforms/javascript/common/session-replay/index.mdx

```diff
@@ -48,6 +48,49 @@ To set up the integration, add the following to your Sentry initialization. Ther
 
 <PlatformContent includePath="session-replay/setup" />
 
+<PlatformSection supported={["javascript"]} notSupported={["javascript.deno", "javascript.bun", "javascript.wasm", "javascript.capacitor", "javascript.electron"]}>
+### Canvas Recording
+
+<Alert level="">
+  Canvas recording is currently in beta. Please submit an issue on
+  [GitHub](https://github.com/getsentry/sentry-javascript) if you encounter any
+  bugs.
+</Alert>
+
+<Alert level="warning">
+  There is currently no PII scrubbing in canvas recordings!
+</Alert>
+
+If you want to record HTML canvas elements, you'll need to add an additional integration in your Sentry configuration. The canvas integration is exported from the browser SDK, so no additional package is required. Canvas recording is opt-in and will be tree-shaken from your bundle if it's not being used:
+
+<PlatformContent includePath="session-replay/setup-canvas" />
```

Comment:
Not sure if this is better? This limits the amount of copy & pasted text by only moving the code snippets to this platform content include.